test/remote/gateways/remote_blue_snap_test.rb

```diff
@@ -6,13 +6,13 @@ def setup
 
     @amount = 100
     @credit_card = credit_card('4263982640269299')
-    @cabal_card = credit_card('6271701225979642', month: 3, year: 2024)
-    @naranja_card = credit_card('5895626746595650', month: 11, year: 2024)
-    @declined_card = credit_card('4917484589897107', month: 1, year: 2023)
-    @invalid_card = credit_card('4917484589897106', month: 1, year: 2023)
+    @cabal_card = credit_card('6271701225979642', month: 3, year: Time.now.year + 1)
+    @naranja_card = credit_card('5895626746595650', month: 11, year: Time.now.year + 1)
+    @declined_card = credit_card('4917484589897107', month: 1, year: Time.now.year + 1)
+    @invalid_card = credit_card('4917484589897106', month: 1, year: Time.now.year + 1)
     @three_ds_visa_card = credit_card('4000000000001091', month: 1)
     @three_ds_master_card = credit_card('5200000000001096', month: 1)
-    @invalid_cabal_card = credit_card('5896 5700 0000 0000', month: 1, year: 2023)
+    @invalid_cabal_card = credit_card('5896 5700 0000 0000', month: 1, year: Time.now.year + 1)
```

Comment:
Why do we need a year here? Can't it just use the default_expiration_date that is provided by `def credit_card` in `test_helper`